x-pack/plugins/canvas/public/lib/create_handlers.ts

```diff
@@ -94,6 +96,7 @@ export const createDispatchedHandlerFactory = (
             break;
           case 'applyFilterAction':
             filters.updateFilter(element.id, event.data);
+            dispatch(fetchAllRenderables());
```

Comment:
Was this a fix for something else? Why was dispatch added?

Reply:
This was part of the circular dependencies fix, since we had to move `fetchAllRenderables` from `updateFilter`